addon/components/uk-button.hbs

```diff
@@ -0,0 +1,20 @@
+<button
+  type={{or @type "button"}}
+  class={{trimmed-join
+    "uk-button"
+    (if @active "uk-active")
+    this.color
+    this.size
+    this.width
+  }}
+  disabled={{@disabled}}
+  title={{@title}}
+  {{on "click" (optional @onClick)}}
```

Comment:
is it still possible to trigger the action by focussing the button and then clicking `Enter`? This works with the previous version, even though I'm not sure why :see_no_evil:

Reply:
Yes this works, this is plain accessibility in browsers.